sdk/ports/_template/az_platform_impl.h

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef _az_PLATFORM_IMPL_H
+#define _az_PLATFORM_IMPL_H
+
+#include <azure/core/az_result.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <azure/core/_az_cfg_prefix.h>
+
+/**
+ * @brief Gets the platform clock in milliseconds.
+ * @remark The moment of time where clock starts is undefined, but if this function is getting
+ * called twice with one second interval, the difference between the values returned should be equal
+ * to 1000.
+ * @return Platform clock in milliseconds.
+ */
+inline AZ_NODISCARD int64_t az_platform_clock_msec() { 
+    #error "go see the docs"
```

Comment:
nit: also in line with previous comment, just to be gentle here and elsewhere in the file
```suggestion
    #error "please see the docs"
```
Should we also have a link to the doc which you are referencing?

Reply:
+1 : there should be a reference (readme.md) within the git repo (assume no Internet connectivity)

Reply:
Yea note I made the mistake of tick tacking on these but his top level comment is
> I'm working on better links to the docs and better comments. Ignore that aspect